frontend/src/Leaderboard/index.jsx

```diff
@@ -20,9 +22,29 @@ const mockedUsers = [
   },
 ];
 
+const styles = StyleSheet.create({
+  searchBar: {
+    width: '100%',
+    borderRadius: 10,
+    backgroundColor: Colors.lightenGrey,
+  },
+  searchBarView: {
+    flexDirection: 'row',
+    margin: 2,
+  },
+});
+
 function Leaderboard() {
+  function mockedSearchUser(username) {
+    // eslint-disable-next-line no-alert
+    alert(`search user ${username}`);
+  }
+
   return (
     <ScrollView>
+      <View style={styles.searchBarView}>
+        <SearchBar style={styles.searchBar} onSubmit={mockedSearchUser} />
```

Comment:
when I click inside the search bar, I think the best user experience is if the icon disappears
refactor the styling, it's too narrow, very hard to click into it, make it a little bigger
also, you can use some placeholder text as well, like 'Search for users' or something

please create a new component for the search bar, we might use it elsewhere in the app!

Reply:
By saying refactor the styling, does it mean we should enlarge the search bar when the user clicks on it?

Reply:
I think it should be bigger by default because it's so small it's hard to click in it on my phone.